src/controllers/Controller.js

```diff
@@ -6,6 +6,10 @@ const secretKey = "1234qwerasdfzxcv";
 const dbRepo = new RUEatsRepository();
 const saltRounds = 10;
 const https = require('https');
+const fs = require('fs');
+const stripe = require('stripe')(
+  'sk_test_51O7NOrDtd64NB5jLGob6J6DhRiHslzfSrNUBhveP8UpK5lKq9ahODHrC4Wicd8JBs56JLu7Bc0vPy0YfXToqJjWL00Bvxgrnh6'
```

Comment:
Is this a token? and is it safe to expose this? Can we move this in .env file

Reply:
Moved to .env